src/sentry/models/release.py

```diff
@@ -228,19 +228,6 @@ def merge(cls, to_release, from_releases):
 
             release.delete()
 
-    @property
-    def short_version(self):
-        return Release.get_display_version(self.version)
-
-    @staticmethod
-    def get_display_version(version):
-        match = _dotted_path_prefix_re.match(version)
-        if match is not None:
-            version = version[match.end() :]
-        if _sha1_re.match(version):
-            return version[:7]
```

Comment:
In the client side changes you're retaining the hash slicing. Would the client code be simpler if the API emitted the 'shortVersion' when the version name was hash shaped?

Reply:
I only kept the hash slicing for commit hashes and not for releases.
We want to get rid of the truncating release names entirely.